kaleidoscopic-kings/backend/card_format.py

```diff
@@ -0,0 +1,316 @@
+import random
+import logging
+from dataclasses import dataclass
+from typing import Union, Tuple, List, Dict, ClassVar
+
+logger = logging.getLogger(__name__)
+
+
+"""
+Changing attributes of any of these data classes requires changing saved json format.
+Purpose of them is to make sure there is no missing data in json aka it will error if json format
+and data class format differ.
+
+Also they are used for the Editor as they will produce the same output format as given input format.
+"""
```

Comment:
A module-level docstring should be placed at the top of the file. This is not only a style convention, but will actually make Python recognize the multiline string as a docstring so it can use it for the output of `help` (among other things).